src/app/core/services/tv2-logger.service.ts

```diff
@@ -0,0 +1,39 @@
+import { DefaultLogger } from '@tv2media/logger/web'
+import { environment } from '../../../environments/environment'
+
+export class Tv2LoggerService extends DefaultLogger {
+  constructor() {
+    ensureWindowEnv({
+      ENV: getExecutionEnvFromUrl() ?? (environment.production ? 'production' : 'development'),
```

Comment:
Are we going to let staging count as 'development'?

Reply:
If nothing else is specified, I think it is a valid strategy. The question should probably be if we want to be able to persist modifications to the execution environment or log level in local storage.

Reply:
Modifications like, what loglevel is being used?

Reply:
Yes.